.travis.yml

```diff
@@ -27,12 +31,17 @@ addons:
       - libstdc++6:i386
       - python
       - python-pip
-  
+
+
 install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $NODE_VERSION
```

Comment:
Should this be a shallow clone with `--depth 1`?  Or is that not possible with the `git describe` thing?